src/utils/setting_handler.py

```diff
@@ -91,6 +91,13 @@ def get_setting(
     :default: If True, returns the default SettingValue when no journal specific
         value is present
     """
+    if not isinstance(setting_name, str):
+        import inspect
```

Comment:
Can the import statement be shifted to the header?

Reply:
Moved.

I wasn't sure if you wanted to keep this sort of workaround, so I kept everything close for easier deletion :slightly_smiling_face: